database/seeds/CentreUsersSeeder.php

```diff
@@ -18,10 +18,22 @@ public function run()
             "password" => bcrypt('store_pass'),
             "role" => "centre_user",
         ]);
+
+        // Attach an initial centre
+        $user1->centres()->attach([
+            1 => ['homeCentre' => true]
+        ]);
+
+        // Get the first centre's sponsor and make two more centres with the same sponsor
+        $sponsor_id = $user1->centres()->first()->sponsor()->first()->id;
+
+        $local_centre1 = factory(App\Centre::class)->create(['sponsor_id' => $sponsor_id]);
```

Comment:
Looks good. You can also do this:
`$local_centres = factory(App\Centre::class, 2)->create(['sponsor_id' => $sponsor_id]);`

and access with:
`$local_centres[0]` and `$local_centres[1]`

Reply:
@katjam This is much neater, I have updated to this.